.github/workflows/maven.yaml

```diff
@@ -7,20 +7,13 @@ on:
    
 jobs:
   build:
-    runs-on: ${{matrix.os}}
+    runs-on: ubuntu-latest
     strategy:
       matrix:
-        # test against several OSes
-        os: [ubuntu-latest, macos-latest, windows-latest]
         # test against several Java versions
         java: [8,11]
```

Comment:
This is the example from the TDD-CI book (could be upgraded with latest Java version 14 and/or 17)